docs/DEVGUIDE.md

```diff
@@ -45,7 +45,7 @@ To clone the repository:
 First `cd` to the root of the cloned repository tree.
 To build the service-catalog you have two options:
 * `make build`
-* `DOCKER=1 make build`
+* `build/run.sh make build`
```

Comment:
this is one of the reasons I would prefer that we don't do this. I'm not a fan of requiring people to put build/run.sh in front of everything, its not intuitive for people. Granted I'm not thrilled with DOCKER=1 either, but that's less objectionable because using env vars as flags is normal for people.  I would much prefer to just default to using Docker - perhaps switch DOCKER=1 with NO_DOCKER=1